CONTINUITY.md

```
@@ -1,54 +1,56 @@
-Hacer merge del PR #187.
+Merge de este PR.
 
 Criterio de avance:
-main sincronizado con el fix. Producción actualizada con migrate + build.
+main con el fix de workspace. Fresh install muestra todos los shortcuts.
 
 ---
 
 ## Estado actual
 
 ### Ya cerrado
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)